prow/config/config.yaml

```diff
@@ -232,6 +233,21 @@ periodics:
             secretName: github-token
 
 presubmits:
+  Nordix/metal3-dev-tools:
+  - name: shellcheck
```

Comment:
yaml lists need to be intended by two spaces. Please indent all lists (lines starting with `-`) by two from previous indent level.

Reply:
so whats our yamllint is checking then?

Reply:
It is only configured to check for trailing spaces, duplicate keys, and truthy values = it doesn't, the config sucks. But the next guy who wants to change the yaml who has yaml formatter configured in their editor will have huge diff.

Vscode has nice yaml plugin from RH you should absolutely install if you work with yaml files btw.

Reply:
Thanks, done

Reply:
I will push changes to yamllint workflow.